### PR TITLE
docs: new structure

### PR DESCRIPTION
**Pull request by soyeric128**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

This PR changes the documentation structure.

Closes #issue

### CODE REVIEW

File: docs/doc/10-deploy/07-query/20-query-metrics.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ description:
   Databend Query Metrics
 ---
 
-A `databend-query` server records metrics in table [system.metrics](../../30-reference/30-sql/70-system-tables/system-metrics.md).
+A `databend-query` server records metrics in table [system.metrics](../../13-sql-reference/70-system-tables/system-metrics.md).
 
-The [metric_api_address](../../50-manage/01-query/10-query-config.md) to listen on that can be scraped by Prometheus and will return a [Prometheus](http://prometheus.io/docs/instrumenting/exposition_formats/) format of metrics.
+The [metric_api_address](../07-query/10-query-config.md) to listen on that can be scraped by Prometheus and will return a [Prometheus](http://prometheus.io/docs/instrumenting/exposition_formats/) format of metrics.
```
Comment (Xuanwo) [inline docs/doc/10-deploy/07-query/20-query-metrics.md:10]:
```suggestion
The [metric_api_address](../07-query/10-query-config.md) to listen on that can be scraped by Prometheus and will return a [prometheus](http://prometheus.io/docs/instrumenting/exposition_formats/) format of metrics.
```

Comment (Xuanwo) [reply docs/doc/10-deploy/07-query/20-query-metrics.md:10]: Trigger build

**Comment by Xuanwo** [comment]
```shell


Frequent broken links are linking to:
--
10:26:47.255 | - /doc/learn/analyze-ontime-with-databend-on-ec2-and-s3
10:26:47.255 |  
10:26:47.255 | Exhaustive list of all broken links found:
10:26:47.255 |  
10:26:47.255 | - On source page path = /doc/deploy/deploying-databend:
10:26:47.255 | -> linking to /doc/reference/sql/ddl/user/user-create-user
10:26:47.255 |  
10:26:47.255 | - On source page path = /doc/develop/nodejs:
10:26:47.256 | -> linking to /doc/reference/sql/ddl/user/user-create-user
10:26:47.256 | -> linking to /doc/reference/sql/ddl/user/grant-privileges
10:26:47.256 |  
10:26:47.256 | - On source page path = /doc/develop/python:
10:26:47.256 | -> linking to /doc/reference/sql/ddl/user/user-create-user
10:26:47.256 | -> linking to /doc/reference/sql/ddl/user/grant-privileges
10:26:47.256 |  
10:26:47.256 | - On source page path = /doc/faq/how-to-benchmark-with-hyperfine:
10:26:47.257 | -> linking to ../learn/analyze-ontime-with-databend-on-ec2-and-s3 (resolved as: /doc/learn/analyze-ontime-with-databend-on-ec2-and-s3)
10:26:47.257 |  
10:26:47.257 | - On source page path = /doc/load-data/stage:
10:26:47.257 | -> linking to /doc/reference/sql/ddl/stage/ddl-create-stage
10:26:47.257 |  
10:26:47.257 | - On source page path = /doc/performance/cvm-cos-performance:
10:26:47.258 | -> linking to /doc/learn/analyze-ontime-with-databend-on-ec2-and-s3
10:26:47.258 |  
10:26:47.258 | - On source page path = /doc/performance/ec2-s3-performance:
10:26:47.258 | -> linking to /doc/learn/analyze-ontime-with-databend-on-ec2-and-s3
10:26:47.258 |  
10:26:47.259 | - On source page path = /doc/performance/ec2-wasabi-performance:
10:26:47.259 | -> linking to /doc/learn/analyze-ontime-with-databend-on-ec2-and-s3
10:26:47.259 |  
10:26:47.259 | - On source page path = /doc/performance/ecs-oss-performance:
10:26:47.259 | -> linking to /doc/learn/analyze-ontime-with-databend-on-ec2-and-s3
10:26:47.259 |  
10:26:47.259 | - On source page path = /doc/performance/qingcloud-qingstore-performance:
10:26:47.260 | -> linking to /doc/learn/analyze-ontime-with-databend-on-ec2-and-s3
10:26:47.260 |  
10:26:47.260 | - On source page path = /doc/performance/scw-os-performance:
10:26:47.260 | -> linking to /doc/learn/analyze-ontime-with-databend-on-ec2-and-s3
10:26:47.260 |  
10:26:47.260 | - On source page path = /doc/sql-commands/query-syntax/dml-json-path:
10:26:47.261 | -> linking to /doc/reference/data-types/data-type-semi-structured-types
10:26:47.261 |  
10:26:47.261 | - On source page path = /doc/sql-reference/data-types/data-type-nullable-types:
10:26:47.261 | -> linking to /doc/reference/functions/conditional-functions/isnull
10:26:47.261 | -> linking to /doc/reference/functions/conditional-functions/isnotnull


```

**Comment by wubx** [comment]
LGTM

**Review by Xuanwo — APPROVED** [review]
Let's Go!

**Pull request merged**